### ISSUE TITLE
preferredNames for one GND entity not in sync

### ISSUE THREAD

**Issue by acka47** [lobid/lodmill#760]
See e.g. http://lobid.org/resource/BT000003404/about (snippet):

``` javascript
{
    "@id" : "http://d-nb.info/gnd/4115393-5",
    "preferredName" : "Niedersachsen, Süd",
    "preferredNameForThePlaceOrGeographicName" : "Niedersachsen"
  }
```

`preferredName`and `preferredNameForThePlaceOrGeographicName` should be identical.

**Comment by dr0i** [comment]
There is only one label in API 2.0, see http://lobid.org/resources/BT000003404.

**Comment by acka47** [comment]
This is an API 1.0 issue. Won't fix in 1.0. Closing.
